src/api/handlers/channel.ts

```diff
@@ -467,8 +467,9 @@ export async function editChannel(
 
   const payload = {
     ...options,
+    reason,
     // deno-lint-ignore camelcase
-    rate_limit_per_user: options.slowmode,
+    rate_limit_per_user: options.rateLimitPerUser,
```

Comment:
Here you can use the camel to snake case function.

Reply:
If only it existed. As I said, that PR was reverted. :sweat_smile:

Reply:
Again? lol why? All tests are passing your/gh deno is weird.

Reply:
> Again? lol why? All tests are passing your/gh deno is weird. Even on my linux machine tests are passing.

No idea, but it won't be merged unless the official GitHub checks are passing, that's for sure.

Reply:
I give up as long as I don't know whats the problem, I cannot solve it.

Reply:
> I give up as long as I don't know whats the problem, I cannot solve it.

Anything I can help with? If not, that's fine too

Reply:
If tests are failing deno should log what went wrong. So in this case what parameter was expected and what deno got. Would be helpful if you can send me that.